### ISSUE TITLE
Visually reflect CV on Octave

### ISSUE THREAD

**Issue by ilambert** [VCVRack/Fundamental#93]
Is there a possibility of visually reflecting the octave set by CV input on the octave module?

It's just something small I thought would be nice.

--Ian

**Comment by AndrewBelt** [comment]
Yes, this is the correct place for Fundamental feature requests. 

How would your proposal work? Given an arbitrary voltage, what would be displayed?

**Comment by ilambert** [comment]
I would think some sort of fainter LED indicator than the one for the currently-selected octave. Something like this?
![Screenshot from 2019-10-10 09-38-14](https://user-images.githubusercontent.com/1820726/66579256-2608f180-eb42-11e9-92a4-2fcbf6f3eedc.png)

**Comment by AndrewBelt** [comment]
This is the behavior of VCV Octave since https://github.com/VCVRack/Fundamental/commit/b9312c8038135dc6a8e6f428b52446a5e850e05e
